packages/instantsearch.js/src/components/Template/Template.tsx

```diff
@@ -110,7 +110,11 @@ See: https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/js/#up
 
     // This is to handle Hogan templates with Fragment as rootTagName
     if (RootTagName === Fragment) {
-      return <RawHtml content={content} key={content} />;
+      return content.trim().charAt(0) === '<' ? (
+        <RawHtml content={content} key={content} />
```

Comment:
Technically this _could_ be a breaking change for a hogan template like this: `Hello <button>world</button>` which right now will render as html, but after this change it will no longer do that I believe. Do you mind adding a test to see if it gets escaped or not (and then check what the behaviour before this PR is).

Reply:
Ah yes it would break those templates...
Even checking if there is any HTML and passing it to `innerHTML` wouldn't work, whatever text node before or after gets discarded.
I'll check for another solution, otherwise plan B would be to put `<Banner>` for the empty template on `instantsearch.js`'s side.

Reply:
Not too sure what the underlying reason of this change is. could we wrap the banner in a div instead of rendering a fragment maybe?

Reply:
This is fixed and I added tests for that